src/test/java/com/cydeo/runners/CucumberRunner.java

```diff
@@ -9,12 +9,13 @@
         plugin = {
                 "html:target/cucumber-reports.html",
                 "rerun:target/rerun.txt",
-                "me.jvt.cucumber.report.PrettyReports:target/cucumber"
```

Comment:
Line 12 is a conflict. I think the issue here is that the project is trying to duplicate this line. Can you try deleting it, and we can see if that fixes it?